File: src/main/kotlin/com/stannis/dataModel/complexStatementTypes/FunctionCallsWithDeclaration.kt
package com.stannis.dataModel.complexStatementTypes

import com.stannis.dataModel.Statement
import com.stannis.dataModel.statementTypes.FunctionCalls

data class FunctionCallsWithDeclaration(override val `$type`: String? = "FunctionCallsWithDeclaration",
                                        var functionCalls: FunctionCalls?,
                                        var declWithParent: Statement?
                                        ):Statement

File: Interpreter/Interpreter/Models/serialize/complexStatementTypes/FunctionCallsWithDeclaration.cs
using Interpreter.Models.serialize.statementTypes;

namespace Interpreter.Models.serialize.complexStatementTypes{
    public class FunctionCallsWithDeclaration : IStatement
    {   
        public FunctionCalls functionCalls { set; get; }
        
        public IStatement declWithParent { set; get; }
    }
}